tst/testinstall/syntaxtree.tst

```diff
@@ -2854,6 +2861,7 @@ rec(
   stats := rec(
       statements := [ rec(
               obj := rec(
+                  string := "1.0",
                   type := "T_FLOAT_EXPR_EAGER",
                   value := 1 ),
```

Comment:
Hmm, how come the value is `1` here? I think that's wrong -- we probably are returning the index into the `values` array, instead of the value stored in it.

Reply:
OK, so it is the value, but again: how is that possible??? I still suspect something is wrong here...

Reply:
OK, so the problem is the print method for floats:
```
gap> Print(1.);
1
```